pyspark/bigdl/models/rnn/README.md

```diff
@@ -103,6 +103,32 @@ PYSPARK_DRIVER_PYTHON=./venv/bin/python PYSPARK_PYTHON=./venv.zip/venv/bin/pytho
 * `--dampening` dampening for momentum, the default value is 0.
 * `--maxEpoch` max number of epochs to train, the default value is 30.
 
+#####In order to use MKL-DNN as the backend, you should:
+1. Define a model with Model(graph container). For RNN models, currently we have MKL-DNN support 
+   for LSTM and GRU.
+2. Specify the input and output formats of it.
+   For example:
+   ```
+   theDefinedModel.set_input_formats([theInputFormatIndex])
+   theDefinedModel.set_output_formats([theOutputFormatIndex])
+   ```
+   BigDL needs these format information to build IRGraph from StaticGraph for MklDnn computing.
```

Comment:
`these formats to build a graph running with MKLDNN backend`

Reply:
Changed

Reply:
doc is not changed

Reply:
Doc changed .